src/utils/events/event-manager.js

```diff
@@ -91,6 +94,13 @@ export default class EventManager {
           this.aliasedEventHandlers[event] = aliasedEventHandler;
         }
       }
+      if (!this.wheelInput && WheelInput.isSourceOf(event)) {
```

Comment:
Nit: To keep code as clean as possible (reduce number of if statements) I would have preferred if the `isSourceOf` thing was internal to these classes - e.g. we'd always create these but they would only register events if needed.

In fact it seems to me that the classes already support a third parameter that does loop over events and only register what was asked for, would it be possible to use that?